### ISSUE TITLE
How can we make the pedestrians obey the traffic rules?

### ISSUE THREAD

**Issue by radiateboy** [eclipse/sumo#7533]
I'm running a scene of pedestrians crossing the road. Why don't pedestrians obey the traffic rules? How can we make the pedestrians obey the traffic rules?
![20200910155114](https://user-images.githubusercontent.com/9248754/92697175-853c3b00-f37d-11ea-9ec6-cb685285c87c.png)

**Comment by namdre** [comment]
They usually do. It's hard to tell whats going on in your scenario
- you've cut of the version information in your screenshot so I cannot tell if this might be due to an old bug
- who knows what kind of settings you did to the pedestrian type. You can configure them to disobey rules
- your tls layout looks weird: the straight traffic doesn't even have a signal. For all I know you might be loading a network with invalid traffic rules
- are you using traci maybe?

In summary: Tell which version you are using and provide input files for reproducing the issue along with your screenshot.

**Comment by radiateboy** [comment]
@namdre okey. 
Maybe it's my traffic signal setting problem.
I'll rebuild try again.  thanks friend.
